mappings/net/minecraft/sound/SoundEvents.mapping

```diff
@@ -1 +1,11 @@
 CLASS net/minecraft/unmapped/C_jtkcimte net/minecraft/sound/SoundEvents
+	FIELD f_jkyfydrb ITEM_GOAT_HORN_SOUND_VARIANTS Lcom/google/common/collect/ImmutableList;
```

Comment:
```suggestion
	FIELD f_jkyfydrb GOAT_HORN_SOUND_VARIANTS Lcom/google/common/collect/ImmutableList;
```

Reply:
The problem is that it breaks with the convention established by the other entries; I do, however, want to discard the convention here, but I don't know what to do

Reply:
`ITEM_GOAT_HORN_VARIANTS`?

Reply:
`sound` is part of the ID though

Reply:
> The problem is that it breaks with the convention established by the other entries; I do, however, want to discard the convention here, but I don't know what to do

I see, I see. I wouldn't personally have a problem with the convention being discarded here, depends what the other think

Reply:
oh well, I'll try your suggestions